### DISCUSSION TITLE
请问下在vuepress2.0中，Home页面的action为何不会生效

### DISCUSSION THREAD

**Discussion by ahehaoyu** [vuepress/core#854] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/43852911/165666851-1f67b1c8-d536-44fb-a156-099592a94e4b.png)  

**参考1.0版本的README配置，不会有action按钮**

![image](https://user-images.githubusercontent.com/43852911/165667028-f7da3d15-ad2e-41bd-a224-7ca56e49fc0c.png)

**Comment by meteorlxy** [comment, marked as answer]
因为支持多个按钮，改成 actions 了。请先阅读 v2 文档，不要拿着 v1 的文档配 v2 

https://vuepress.github.io/reference/default-theme/frontmatter.html#actions
